composer/plan.sh

```diff
@@ -16,7 +16,11 @@ do_unpack(){
 }
 
 do_build() {
-  return 0
+  cat<<EOF > "$pkg_prefix/bin/composer"
```

Comment:
This needs to go in `do_install` because the `$pkg_prefix` directories do not exist during `do_build`.